CHANGES.md

```diff
@@ -159,4 +155,4 @@
 
 ### v0.1, 2018-05-14
 
-* initial release
+* initial released
```

Comment:
Is there a particular reason to introduce this grammatical error?

Reply:
It feels Hacktoberfest related (https://twitter.com/dan_abramov/status/1311692006775758853)